projects/templates/plopfile.js

```diff
@@ -38,14 +38,23 @@ module.exports = function (plop) {
         const capitalized = camel.charAt(0).toUpperCase() + camel.substring(1);
         return capitalized;
     });
+
     plop.setActionType('install deps', function (answers) {
         execSync(
             `cd ../../ && yarn lerna add @spectrum-web-components/base --scope=@spectrum-web-components/${answers.name} --no-bootstrap`
         );
+        if (answers.spectrum)
+            execSync(
+                `cd ../../ && yarn lerna add @spectrum-css/${answers.spectrum} --scope=@spectrum-web-components/${answers.name} --dev --no-bootstrap`
+            );
+    });
+
+    plop.setActionType('format files', function (answers) {
         execSync(
-            `cd ../../ && yarn lerna add @spectrum-css/${answers.spectrum} --scope=@spectrum-web-components/${answers.name} --dev --no-bootstrap`
+            `cd ../../ && yarn prettier --write packages/${answers.name} && eslint --fix -f pretty packages/${answers.name} && stylelint --fix packages/${answers.name}`
```

Comment:
This will add the license to the files - will be nice to do it this way too because if the license ever changes, it's coming from the same source of truth.